catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -1251,7 +1251,7 @@ protected SpatialSearch createCombinedSpatialSearch(
         for (PointRadius search : pointRadiusSearches) {
           BoundingBox bbox = BoundingBoxUtils.createBoundingBox(search);
           List bboxCoordinate = BoundingBoxUtils.getBoundingBoxCoordinatesList(bbox);
-          List<List> coordinates = new ArrayList<>();
+          List<List<Double>> coordinates = new ArrayList<>();
           coordinates.add(bboxCoordinate);
```

Comment:
I feel like I'm missing something here... If `BoundingBoxUtils.getBoundingBoxCoordinatesList(bbox)` returns a `List<List<Double>>`, then shouldn't `coordinates` be `List<List<List<Double>>>`?

Reply:
That seems right, i might have messed that up.  This is why the lists should be typed in the first place.